python/Makefile

```diff
@@ -49,13 +49,13 @@ style:
 	isort --apply --recursive $(STYLE_TARGETS) --skip-glob "$(EXCLUDE_TARGETS)/*"
 	autoflake -i --remove-all-unused-imports -r $(STYLE_TARGETS) --exclude "$(EXCLUDE_TARGETS)"
 	flake8
-	pyright
+	make pyright
 
 style_check:
 	black --check $(STYLE_TARGETS)
 	isort --check-only --recursive $(STYLE_TARGETS) --skip-glob "$(EXCLUDE_TARGETS)/*"
 	flake8
-	pyright
+	make pyright
```

Comment:
Nit: can also be listed as dependency after `style_check:` if you don't mind it running first.

Reply:
How would it look and work like? Just deleting this line and specifying `style_check: pyright` would mean the `make pyright` logic would be running as a first stage (before the `black --check`)?

I would still leave it at the end however, as we always run the `black` and `isort` tests first

Reply:
That is correct: https://makefiletutorial.com/#makefile-syntax The page talks about files but it works for target names even if they don't produce any file. (Well, this breaks if something creates file named `pyright` in the directory. Can be avoided if you mark the target as [.PHONY](https://makefiletutorial.com/#phony)).

I'm OK with leaving at the end.